Sources/Basic/TerminalController.swift

```diff
@@ -78,11 +78,13 @@ public final class TerminalController {
             return width
         }
 
+#if TIOCGWINSZ
```

Comment:
I suspect that might be the case.  It looks like powerpc defines TIOCGWINSZ using the _IOR macro to come up with a unique ioctl ID, whereas on x86 it's hardcoded as 0x5413.  Is there somewhere within Swift that could read ioctls defined with _IO/_IOR/_IOW/_IORW and export them in a way that Swift code can use them?